sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,76 +1,114 @@
 # Release History
 
+## 15.0.0 (2021-06-01)
+
+### Features
```

Comment:
```suggestion
### Features Added
```

Reply:
The change log format is documented in [these guidelines](https://azure.github.io/azure-sdk/policies_releases.html#change-logs).   It's not worth going back and updating retroactively, but we should adjust the current version.